### PR TITLE
Add keybinds modmenu, make camera move faster

### PR DESCRIPTION
**Pull request by ygsbzr**

### CODE REVIEW

**Comment by BasedJellyfish11** [comment]
I don't actually maintain this anymore so I'll leave the choice up to @fifty-six, but increasing the camera movement speed this much will reduce precision which is the true goal of the project, so it's not something I agree with. If you want it to move faster, you can double tap the key to run the coroutine twice at the same time.

File: ScreenshotMachine/CameraHandler.cs [hunk 1]
```diff
@@ -17,18 +17,18 @@ public static class CameraHandler
...
                 GameCameras.instance.cameraController.FreezeInPlace();
                 _frozenCamera = true;
             }
 
-            while (Input.GetKey(keyPressed))
+            while (action.IsPressed)
             {
                 GameCameras.instance.cameraController.transform.position += movement;
-                yield return new WaitForSeconds(0.1f);
+                yield return new WaitForSeconds(0.005f);
```
Comment (fifty-six) [inline ScreenshotMachine/CameraHandler.cs]: I'd prefer this stay mostly unchanged, if you want to add speed something like ```cs
yield return new WaitForSeconds(Input.GetKey(Shift) ? 0.005f : 0.1)
```
would be preferable as the precision is very much wanted when taking screenshots.

File: ScreenshotMachine/Settings.cs [hunk 2]
```diff
@@ -4,65 +4,55 @@
...
-        public KeyCode BlurPlaneToggleKey = KeyCode.R;
-        [FormerlySerializedAs("keyToggleCenterLine")]
-        [JsonConverter(typeof(StringEnumConverter))]
-        public KeyCode ToggleCenterLineKey = KeyCode.E;
-        
-        [FormerlySerializedAs("keyRestoreCamera")]
-        [JsonConverter(typeof(StringEnumConverter))]
-        public KeyCode RestoreCameraKey = KeyCode.Q;
+        [JsonConverter(typeof(PlayerActionSetConverter))]
+        public KeyBinds keyBinds = new KeyBinds();
```
Comment (fifty-six) [inline ScreenshotMachine/Settings.cs]: Would prefer this PascalCase'd as it's C#, maybe [SerializedAs("keyBinds")]

File: ScreenshotMachine/ScreenshotMachine.cs [hunk 3]
```diff
@@ -33,6 +33,8 @@ public override void Initialize()
         public new static void Log(object message) => Logger.Log("[ScreenshotMachine] - " + message );
 
         public override string GetVersion() => VersionUtil.GetVersion<ScreenshotMachine>();
+        public bool ToggleButtonInsideMenu => false;
```
Comment (fifty-six) [inline ScreenshotMachine/ScreenshotMachine.cs:37]: Need an empty line between members

**Review by fifty-six — CHANGES_REQUESTED** [review]

**Comment by fifty-six** [comment]
This looks v nice, thanks. Commented a bit on some minor nits and something regarding the speed stuff.

File: ScreenshotMachine/ModMenu.cs [hunk 4]
```diff
@@ -0,0 +1,33 @@
+﻿using Satchel.BetterMenus;
+namespace ScreenshotMachine
+{
+    public class ModMenu
+    {
+        private static Menu MenuRef;
+        public static MenuScreen GetMenu(MenuScreen lastmenu)
+        {
+            if (MenuRef == null)
+            {
```
Comment (fifty-six) [inline ScreenshotMachine/ModMenu.cs]: small nit bit the style in this project is no braces for 1 line stuff

**Review by fifty-six — APPROVED** [review]

**Comment by fifty-six** [comment]
Thank you!

**Pull request merged**